frontend/webapp/reuseable-components/code/index.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from 'styled-components';
+import { Highlight, themes as prismThemes } from 'prism-react-renderer';
+import { flattenObjectKeys, safeJsonParse, safeJsonStringify } from '@/utils';
+
+interface Props {
+  language: string;
+  code: string;
+  flatten?: boolean;
+}
+
+const Token = styled.span`
+  white-space: pre-wrap;
+  overflow-wrap: break-word;
+  opacity: 0.75;
+  font-size: 12px;
+  font-family: ${({ theme }) => theme.font_family.code};
+`;
+
+export const Code: React.FC<Props> = ({ language, code, flatten }) => {
+  const str = flatten && language === 'json' ? safeJsonStringify(flattenObjectKeys(safeJsonParse(code, {}))) : code;
```

Comment:
we can move this outside of the component o make it more ruseable

Reply:
This is the place to make it re-usable, placing it outside would then require copy-pasting of this logic throughout other files.